src/playground/services/invoices/retrieve/view.clj

```diff
@@ -3,14 +3,14 @@
             [playground.views :as views]))
 
 
-(defn update-invoice [user]
+(defn update-invoice [entry]
```

Comment:
I'm not sure the renaming is an improvement. `entry` could mean anything, which in practice means I don't know what this function argument is about.

Reply:
There's some in-ambiguity now with the naming as there are two tables, one for actual users entering a session (username, password) and the other is the db users table which is random (name, email). That led me to use the term entry for an entry in the `users` table, but I'll revert it and we see later if it becomes unclear perhaps we need to change the naming or the theme.